components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -170,6 +170,7 @@ export default function TeamSettingsPage() {
                         All <b>members</b> of this organization will lose access to this organization, associated
                         projects and workspaces.
                     </li>
+                    <li className="ml-5">Any free credit allowances granted to this organization will be lost.</li>
```

Comment:
**nitpick**: Minor copy suggestion—feel free to pick up parts of it or leave it as is.

```suggestion
                    <li className="ml-5"Any remaining <b>credits</b> will be lost.</li>
```

Reply:
I think the wording of any free credit allowances is a bit more explicit and intentional in describing that you'll lose your monthly free credit allowance if this org has one (i.e. the first org upon joining).

Reply:
Fair points, @selfcontained. However, the credits we may grant to organizations may not be given for free but I trust you as a natural language speaker here. 🙇